addons/l10n_nz/data/account_tax_report_data.xml

```diff
@@ -42,7 +42,7 @@
         <field name="name">[BOX 8] Multiply the amount in Box 7 by 3 and then divide by 23</field>
         <field name="parent_id" ref="tax_report_sale_and_income"/>
         <field name="sequence" eval="4"/>
-        <field name="formula">((NZBOX5 - NZBOX6) * 3)/23</field>
+        <field name="formula">(NZBOX7 * 3)/23</field>
```

Comment:
Where is `NZBOX7` defined? You are missing the `<field name="code">NZBOX_</field>` nodes

Reply:
:person_facepalming: